src/Manifolds.jl

```diff
@@ -165,9 +165,29 @@ include("groups/rotation_action.jl")
 
 include("groups/special_euclidean.jl")
 
-
 include("tests/ManifoldTests.jl")
 
+@doc raw"""
+    p ∈ M
+    Base.in(p,M; kwargs...)
```

Comment:
I would put `Base.in` in the first line of the docstring, similarly to the docstring in `Base`.